ac/ac-chat/src/Chat.jsx

```diff
@@ -70,11 +70,6 @@ const Chatmsg = ({ msg }) => (
 export default class Chat extends Component<ActivityRunnerT> {
   node: any;
 
-  constructor(props: Object) {
```

Comment:
@lfaucon conceptually when do we need to bind this to the class methods? Going through React's documentation, I assumed that whenever you have a method, you bind this to it in the constructor. 

Is it when you use it inside the template? for eg: <input onChange={this.handleChange} />

Reply:
Using arrow functions removes the need to bind `this`. https://www.sitepoint.com/bind-javascripts-this-keyword-react/